src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -4759,154 +4846,233 @@ namespace SystemAvailabilityManager {
-					} else {
-						ShowRecurringWarningErrorAtEnd( "Hybrid ventilation control: " + HybridVentSysAvailMgrData( SysAvailNum ).AirLoopName + " No humidistat control impact continues...", HybridVentSysAvailMgrData( SysAvailNum ).DewPointErrIndex, double( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode ), double( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode ) );
+					found = false;
+					for ( HStatZoneNum = 1; HStatZoneNum <= NumHumidityControlZones; ++HStatZoneNum ) {
+						if ( HumidityControlZone( HStatZoneNum ).ActualZoneNum == ZoneNum ) {
+							found = true;
+							ZoneRHHumidifyingSetPoint = GetCurrentScheduleValue( HumidityControlZone( HStatZoneNum ).HumidifyingSchedIndex );
+							ZoneRHDehumidifyingSetPoint = GetCurrentScheduleValue( HumidityControlZone( HStatZoneNum ).DehumidifyingSchedIndex );
+							if ( ZoneAirRH > ZoneRHDehumidifyingSetPoint ) { // Need dehumidification
+								WSetPoint = PsyWFnTdbRhPb( MAT( ZoneNum ), ( ZoneRHDehumidifyingSetPoint / 100.0 ), OutBaroPress );
+								if ( WSetPoint < OutHumRat ) HybridVentSysAvailMgrData( SysAvailNum ).VentilationCtrl = HybridVentCtrl_Close;
+							} else if ( ZoneAirRH < ZoneRHHumidifyingSetPoint ) { // Need humidification
+								WSetPoint = PsyWFnTdbRhPb( MAT( ZoneNum ), ( ZoneRHHumidifyingSetPoint / 100.0 ), OutBaroPress );
+								if ( WSetPoint > OutHumRat ) HybridVentSysAvailMgrData( SysAvailNum ).VentilationCtrl = HybridVentCtrl_Close;
+							} else {
+								HybridVentSysAvailMgrData( SysAvailNum ).VentilationCtrl = HybridVentCtrl_Close;
+							}
+						}
+					}
+					if ( ! found && NumHumidityControlZones > 0 ) {
+						++HybridVentSysAvailMgrData( SysAvailNum ).DewPointErrCount;
+						if ( HybridVentSysAvailMgrData( SysAvailNum ).DewPointErrCount < 2 ) {
+							ShowWarningError( "Hybrid ventilation control: The zone for dew point control mode is different from the zone for ZoneControl:Humidistat=" + HybridVentSysAvailMgrData( SysAvailNum ).AirLoopName );
+							ShowContinueError( "The Zone name for hybrid control is " + Zone( ZoneNum ).Name + ". Humidistat has no impact" );
+							ShowContinueError( "HVAC system may turn off when outdoor dewpoint is between min and max dewpoint." );
+							ShowContinueErrorTimeStamp( "" );
+						} else {
+							ShowRecurringWarningErrorAtEnd( "Hybrid ventilation control: " + HybridVentSysAvailMgrData( SysAvailNum ).AirLoopName + " No humidistat control impact continues...", HybridVentSysAvailMgrData( SysAvailNum ).DewPointErrIndex, double( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode ), double( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode ) );
+						}
 					}
 				}
-			}
 
-			// Outdoor ventilation air control mode
-			if ( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode == HybridVentMode_OA ) {
+				// Outdoor ventilation air control mode
+				if ( HybridVentSysAvailMgrData( SysAvailNum ).ControlMode == HybridVentMode_OA ) {
 
+				}
 			}
 		}
 
```

Comment:
If I am following correctly, KeepStatus is overridden by the max wind speed and rain checks.  If so, that should be mentioned in the docs describing the min operation/vent time fields.